app/src/main/java/woowacourse/paint/model/DrawingElement.kt

```diff
@@ -0,0 +1,26 @@
+package woowacourse.paint.model
+
+import android.graphics.Paint
+import android.graphics.Path
+
+data class DrawingElement(
+    val path: Path = Path(),
+    val paint: Paint = Paint(),
+) {
```

Comment:
최상위 클래스라 불변으로 만들생각은 안해봤는데 기초가 탄탄 하시다는 게 느껴지네요!!
하지만 내부 객체들이 불변이 아니기 때문에 완벽한 불변을 위해 이 또한 보안해줘야한다고 생각합니다!
프로젝트 전반적으로 다음과 같이 내부에 직접 접근하지 못하도록 막아보는 것은 어떨까요??

```suggestion
data class DrawingElement(
    private val path: Path = Path(),
    private val paint: Paint = Paint(),
) {
```